lmdeploy/pytorch_poc/patch/chatglm2.py

```diff
@@ -0,0 +1,290 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+# Adapted from https://huggingface.co/THUDM/chatglm2-6b/blob/main/modeling_chatglm.py   # noqa: E501
+
+import importlib
+from typing import Optional, Tuple
+
+import torch
+import torch.nn as nn
+import torch.utils.checkpoint
+from transformers.modeling_outputs import BaseModelOutputWithPast
+from transformers.utils import TRANSFORMERS_DYNAMIC_MODULE_NAME
+
+from lmdeploy.pytorch_poc.kernels import paged_attention_fwd
+
+chatglm_module = importlib.import_module('.chatglm2-6b.modeling_chatglm',
```

Comment:
I can not load the model `THUDM/chatglm2-6b`.

```
ModuleNotFoundError: No module named 'transformers_modules.chatglm2-6b'
```

Reply:
Fixed